packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -38,8 +38,10 @@ BuildRequires: npm(@theforeman/builder) >= 4.14.0
 # end package.json devDependencies BuildRequires
 
 # start package.json dependencies BuildRequires
-BuildRequires: npm(react-intl) >= 2.8.0
-BuildRequires: npm(react-intl) < 3.0.0
+BuildRequires: npm(react-native-elements) >= 3.4.2
+BuildRequires: npm(react-native-elements) < 4.0.0
+BuildRequires: npm(react-native-vector-icons) >= 9.0.0
+BuildRequires: npm(react-native-vector-icons) < 10.0.0
```

Comment:
we don't have those two packaged. we could if they are needed and not provided by `@theforeman/vendor`.

cc @Ron-Lavi

Reply:
however, I do not see any `react-native` references in https://github.com/ATIX-AG/foreman_scc_manager?

Reply:
it builds fine without this locally, so I've opened https://github.com/ATIX-AG/foreman_scc_manager/pull/122

Reply:
I will just remove them. Also thanks for fixing that space issue!

Reply:
in the source repo, yeah.

feel free to just drop the lines here in the spec for now, no need to wait for my PR to be merged and release a new version.

Reply:
@evgeni Now that looks much better! I guess we can just leave the new dev build requirement in there, it won't hurt for production run, will it?